stdlib/REPL/test/TerminalMenus/multiselect_menu.jl

```diff
@@ -57,3 +53,5 @@ end
 # Test SDTIN
 multi_menu = MultiSelectMenu(string.(1:10), charset=:ascii)
 @test simulate_input(Set([1,2]), multi_menu, :enter, :down, :enter, 'd')
+multi_menu = MultiSelectMenu(["single option"])
```

Comment:
You have a CI failure due to tests being run with `--depwarn=error`.

```suggestion
multi_menu = MultiSelectMenu(["single option"], charset=:ascii)
```

and similarly for `RadioMenu` below.

Reply:
Added `charset=:ascii` to other non-legacy tests.

I feel like it's a bit confusing behavior though, especially when we have an exception like `pagesize` kwarg doesn't trigger new interface.

Reply:
I know. I don't think there's another way to get to a sane interface without breaking backwards compatibility. I was extremely pleased to find something so unintrusive, even if it's far from perfect. We can drop the `CONFIG` interface in Julia 2.0, but that will likely be years away.